lti_consumer/lti_consumer.py

```diff
@@ -925,15 +926,37 @@ def _get_context_for_template(self):
 
         # use bleach defaults. see https://github.com/jsocol/bleach/blob/master/bleach/__init__.py
         # ALLOWED_TAGS are
-        # ['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol',  'strong', 'ul']
+        # ['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol',  'strong', 'ul', 'img']
         #
         # ALLOWED_ATTRIBUTES are
         #     'a': ['href', 'title'],
         #     'abbr': ['title'],
         #     'acronym': ['title'],
-        #
+        #     'img': ['src']
+
         # This lets all plaintext through.
-        sanitized_comment = bleach.clean(self.score_comment)
+        allowed_tags = [
+            'a',
+            'abbr',
+            'acronym',
+            'b',
+            'blockquote',
+            'code',
+            'em',
+            'i',
+            'li',
+            'ol',
+            'strong',
+            'ul',
+            'img'
+        ]
+        allowed_attributes = {
+            'a': ['href', 'title'],
+            'abbr': ['title'],
+            'acronym': ['title'],
+            'img': ['src']
```

Comment:
We won't be able to use `alt` attribute. Not sure of the overall usage of this xblock but it can introduce a11y problems.